src/services/myPatients.ts

```diff
@@ -160,9 +158,7 @@ export const fetchMyPatients = async (): Promise<CohortData | undefined> => {
 
       const patientsIds = patients.map((p) => p.id ?? '').filter(Boolean)
       const encounters = getApiResponseResources(
-        await api.get<FHIR_API_Response<IEncounter>>(
-          `/Encounter?subject:Patient=${patientsIds.join(',')}&_count=700${API_RESOURCE_TAG}`
-        )
+        await api.get<FHIR_API_Response<IEncounter>>(`/Encounter?subject=${patientsIds.join(',')}`)
```

Comment:
I'm pretty sure that this is not necessary, you only have access to Encounters you are allowed to see (just like Patients) so you can just request `/Encounter` here